sections/layer2/deposit/DepositTab/DepositTab.tsx

```diff
@@ -93,10 +96,16 @@ const DepositTab = () => {
 				setDepositTxError(null);
 				setTxModalOpen(true);
 
-				const transaction = await SynthetixBridgeToOptimism.deposit(
-					parseEther(transferableCollateral.toString()),
-					{ gasLimit: gasLimitEstimate, gasPrice: normalizedGasPrice(gasPrice) }
-				);
+				const amountToDepositBN = toBigNumber(amountToDeposit);
+
+				if (amountToDepositBN.gt(transferableCollateral)) {
```

Comment:
This displays an error if the entered deposit amount is above the user's current `transferableCollateral`